lua/lib/lattice.lua

```diff
@@ -90,82 +87,95 @@ function Lattice.auto_pulse(s)
     s:pulse()
     clock.sync(1/s.ppqn)
   end
-end 
+end
 
---- advance all patterns in this lattice a single by pulse, call this manually if lattice.auto = false
+--- advance all sprockets in this lattice a single by pulse, call this manually if lattice.auto = false
 function Lattice:pulse()
   if self.enabled then
-    local ppm = self.ppqn * self.meter
+    local ppc = self.ppqn * 4
+    -- prefer "cycle" to "meter"; "4" because a "quarter note" is "1/4"
```

Comment:
i like this renaming. how about we change the comment to
`-- pulses per cycle; "4" because a "quarter note" is "1/4"`